snforge_std/src/event.cairo

```diff
@@ -0,0 +1,206 @@
+use array::ArrayTrait;
+use array::SpanTrait;
+use clone::Clone;
+use serde::Serde;
+use option::OptionTrait;
+use traits::Into;
+
+use starknet::testing::cheatcode;
+use starknet::ContractAddress;
+
+#[derive(Drop, Serde)]
+enum SpyOn {
+    All: (),
+    One: ContractAddress,
+    Multiple: Array<ContractAddress>
```

Comment:
As I mentioned, one and multiple are not exclusive. I would replace it with `selected` instead.

Reply:
Why 🤔